doc/publications/cooling.rst

```diff
@@ -0,0 +1,14 @@
+Cooling
+=======
+
+Deep seawater cooling and desalination: Combining seawater air conditioning and desalination
```

Comment:
Is this the figure caption? If so, see [here](https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure) for how it should be included in the `.. figure:` directive.

Reply:
This was meant to be the heading. The figures don't have captions so far.